docs/BuildOptions.md

```diff
@@ -174,275 +153,236 @@ as the `inet_pton` function gets a poor-man's simple replacement that can
 only resolve numeric IPv4 addresses.
 
 
-`--enable-logging` (default: ON)
--------------------------------
+**`--enable-logging`** (default: ON)
 
-Enables logging. When you turn this option OFF, logging will not be
-done at all. Could be tried when you suspect the logging system of
-impairing the performance, at the expense of not having some information
-about errors happening during runtime.
+Enables logging. When you turn this option OFF, the library will not report
+any runtime information through the logging system, including errors.
+This option may be useful if you suspect the logging system of
+impairing performance.
 
 
-`--enable-monotonic-clock` (default: OFF)
------------------------------------------
-
-Enforced `clock_gettime` with monotonic clock on Garbage Collector CV.
-**This is a temporary fix for #729**: The library could got stuck when you
-modify the system clock while a transmission is done over SRT.
+**`--enable-monotonic-clock`** (default: OFF)
 
 This option enforces the use of `clock_gettime` to get the current
-time, instead of `gettimeofday`. This function allows to force a monotonic
-clock, which is independent on the currently set time in the system. The CV,
-for which the `*_timedwait()` functions are used with so obtained current time,
-must be appropriately configured (and this is done so for now only for the
-GarbageCollector controlling CV, not every CV used in SRT). The consequence of
-enabling this option, however, may be portability issues around the
-`clock_gettime()` function, which is not available on every SDK, or extra
+time, instead of `gettimeofday`. This function forces the use of a monotonic
+clock that is independent of the currently set time in the system.
+The condition variables, for which the `*_timedwait()` functions are used
+with time specification basing on the time obtained from `clock_gettime`
+must be appropriately configured. For now, this is only done for the
+GarbageCollector controlling CV, not every CV used in SRT. The consequence
+of enabling this option, however, may be portability issues resulting from
+the fact that `clock_gettime` function may be unavailable in some SDKs or an extra
 `-lrt` option is sometimes required (this requirement will be autodetected).
 
```

Comment:
I suggest adding this (edited) paragraph from your comments:

What is important here is that the `clock_gettime` function is used to obtain 
the current time, and then the `*_timedwait` functions need to be passed the 
time specification from the same domain. The time must be specified to the 
`*_timedwait` functions as the absolute time up to which they have to wait. 
This means that usually you first obtain the current time from `clock_gettime`, 
add the "waiting time" to it, and then pass the result to the `*_timedwait` function.

Reply:
Ok, I'll try to provide the description a little bit more concise.

Reply:
Maybe this:

The problem is based on the fact that POSIX functions that use timeout
specification (all of `*_timedwait`) expect the absolute time value.
A relative timeout value can be then only specified by adding it to
the current time, which can be specified as either system or monotonic
clock (which one, is configured in the resources used in the operation).
However the current time of the monotonic clock can only be obtained by
the `clock_gettime` function.